### PR TITLE
Avatar badge

### PR DESCRIPTION
**Pull request by krgarima**
This branch contains 2 folders - Avatar and Badge
Please review both.

### CODE REVIEW

**Comment by arulk8** [comment]
@krgarima, when I hover over the navbar elements eg "component" it feels like it jumping. the reason would be you are adding padding on hover pls add the padding for direct class as well
element.style {
}
```
h3 a {
    **padding: 0.2rem;**
    color: var(--primary-color);
    font-size: 102%;
}
```

**Comment by arulk8** [comment]
And resolve the merge conflict as well. the UI looks nice.

File: Components/Avatar/avatar.html [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+    <link rel="stylesheet" href="/Components/Avatar/avatar.css">
+    <script src="https://kit.fontawesome.com/d53bb09bcf.js" crossorigin="anonymous"></script>
+
+    <title>Avatar page</title>
+</head>
+
+<body>
+    <header></header>
+    <main>
+        <div class="Container parentContainer" id="modalDrawerMainBody">
```
Comment (arulk8) [inline Components/Avatar/avatar.html:17]: pls use container.just a naming convesion

File: Components/Avatar/avatar.css [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+  cursor: pointer;
+}
+
+/* common */
+
+.wrap {
+  flex-wrap: wrap;
+}
+
+.h-4 {
```
Comment (arulk8) [inline Components/Avatar/avatar.css:131]: I would suggest giving a more meaningful name. eg heading-1.

**Review by Chiragmodi01 — COMMENTED** [review]
- The image inside the avatar is shrinking, add object-fit to that.
- The badges are overlapping while scrolling down, so give the navbar z-index.
- And there's some branch conflicts.

Besides that, it looks good.

**Review by arulk8 — CHANGES_REQUESTED** [review]

**Pull request merged**